aten/src/ATen/native/TensorIterator.cpp

```diff
@@ -654,6 +654,21 @@ TensorIterator TensorIterator::binary_op(Tensor& out, const Tensor& a,
   return iter;
 }
 
+TensorIterator TensorIterator::comparison_op(Tensor& out, const Tensor& a,
```

Comment:
you may want to call this `logical_op` since we're probably also going to want to port `==` and `!=`.

Reply:
As for me "comparison" is more preferable here. I think "logical" is more about or/and/xor etc. operations